server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/ScheduledTaskLoggerService.java

```diff
@@ -136,30 +145,31 @@ public String getFullLogDownloadUrl(Long scheduleId, Long scheduleTaskId, OdcTas
     }
 
     @SneakyThrows
-    private File getLogFileFromTaskFramework(Long jobId, OdcTaskLogLevel level) {
+    private void consumeLogFromTaskFramework(Long jobId, OdcTaskLogLevel level,
+            Consumer<File> logFileConsumer,
+            Consumer<String> logContentConsumer,
```

Comment:
can not all return input-stream equally, because if the task pod return log content,  for example is:

-------------------
[2023-01-01]log
[2023-01-01]log
[2023-01-01]log
[ODC INFO]: Logs exceed max limitation (3 rows or %s), only the latest part is displayed.Please download the log file for the full content. 
-------------------

then if all return input-stream from task-framework, because pre-function will use `private String getLog(Long scheduleTaskId, OdcTaskLogLevel level)` , the function will use LogUtils.getLatestLogContent(), now I have get the stream from log content, then will get a error log content below:

------------------
[2023-01-01]log
[2023-01-01]log
[ODC INFO]: Logs exceed max limitation (3 rows or %s), only the latest part is displayed.Please download the log file for the full content. 
[ODC INFO]: Logs exceed max limitation (3 rows or %s), only the latest part is displayed.Please download the log file for the full content. 
--------------------

so the task pod content need handle alone, due to this reason I think use `Consumer` is a good method;